proto/cosmos/protocolpool/v1/tx.proto

```diff
@@ -29,6 +29,12 @@ service Msg {
 
   // ClaimBudget defines a method to claim the distributed budget.
   rpc ClaimBudget(MsgClaimBudget) returns (MsgClaimBudgetResponse);
+
+  // CreateContinuousFund defines a method to add funds continuously.
+  rpc CreateContinuousFund(MsgCreateContinuousFund) returns (MsgCreateContinuousFundResponse);
+
+  // CancelContinuousFundProposal defines a method for cancelling continuous fund proposal.
+  rpc CancelContinuousFundProposal(MsgCancelContinuousFundProposal) returns (MsgCancelContinuousFundProposalResponse);
```

Comment:
```suggestion
  rpc CancelContinuousFund(MsgCancelContinuousFundProposal) returns (MsgCancelContinuousFundProposalResponse);
```

I think we should drop the proposal here. Look at `UpdateParams` for instance, it works on an authority so it does not have to be a proposal.